package.json

```diff
@@ -43,11 +43,13 @@
     "url": "https://github.com/fmgrafikdesign/SimpleWebSerialJS/issues"
   },
   "homepage": "https://github.com/fmgrafikdesign/SimpleWebSerialJS#readme",
+  "dependencies": {
+    "@types/w3c-web-serial": "^1.0.7"
+  },
```

Comment:
I moved the typings to the dependencies because the getPort() function in SimpleWebSerial.ts returns a serial port, which is typed as SerialPort which is inferred from the @types/w3c-web-serial package. If the package is not available (e.g. because it's in the dev dependencies), users who install the library do not have all the necessary types installed. What would be a better way to handle this?

Reply:
@coderabbitai I moved the typings to the dependencies because the getPort() function in SimpleWebSerial.ts returns a serial port, which is typed as SerialPort which is inferred from the @types/w3c-web-serial package. If the package is not available (e.g. because it's in the dev dependencies), users who install the library do not have all the necessary types installed. What would be a better way to handle this?